packages/lesswrong/server/dialogueHelperBotCron.ts

```diff
@@ -0,0 +1,3 @@
+import { addCronJob } from './cronUtil';
```

Comment:
Is this PR ready for review?  @lagerros

Reply:
yes @b0b3rt , though I wilil also need to add the textbox as discussed yesterday

Reply:
I mean that this file is empty.  The new cron job is in `dialogues/cron.ts`?